doc/cascadia/profiles.schema.json

```diff
@@ -629,6 +629,11 @@
           "description": "When set to true, tabs are always displayed. When set to false and \"showTabsInTitlebar\" is set to false, tabs only appear after opening a new tab.",
           "type": "boolean"
         },
+        "inputServiceWarning": {
```

Comment:
Nit: consider renaming into "showInputServiceWarning" (here and in other places)

Reply:
I'm gonna say that this one is probably okay as is. The other warnings are `largePasteWarning` and `multiLinePasteWarning`, so this is actually consistent.